src/main/resources/application.properties

```diff
@@ -1,8 +1,15 @@
-spring.datasource.url = jdbc:postgresql://localhost:5432/springdb
-spring.datasource.username = postgres spring.datasource.password=postgres123
-spring.jpa.hibernate.ddl-auto = create-drop
+spring.config.import=credentials.properties
```

Comment:
jesli importujemy jakies propsy to przydalby sie w repo ich wzorzec bez wartosci